pkgs/by-name/wa/wasabibackend/package.nix

```diff
@@ -35,26 +36,29 @@ buildDotnetModule rec {
     zlib
   ];
 
-  preConfigure = ''
-    makeWrapperArgs+=(
-      --chdir "$out/lib/${pname}"
-    )
-  '';
+  executables = [ "WalletWasabi.Backend" ];
 
   postFixup = ''
     mv $out/bin/WalletWasabi.Backend $out/bin/WasabiBackend
   '';
 
-  passthru.tests = {
-    inherit (nixosTests) wasabibackend;
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = {
+      inherit (nixosTests) wasabibackend;
+    };
   };
 
-  meta = with lib; {
+  meta = {
+    changelog = "https://github.com/WalletWasabi/WalletWasabi/releases/tag/${src.tag}";
     description = "Backend for the Wasabi Wallet";
     homepage = "https://wasabiwallet.io/";
-    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    license = licenses.mit;
-    maintainers = with maintainers; [ mmahut ];
+    license = lib.licenses.mit;
+    mainProgram = "WasabiBackend";
+    maintainers = with lib.maintainers; [
+      mmahut
+    ];
     platforms = [ "x86_64-linux" ];
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
```

Comment:
The dependencies are binary code but the package itself is compiled from source, no?